### PR TITLE
docs: update route-preloading-in-angular

### PR DESCRIPTION
**Pull request by Alex61NN5**
Changes proposed in this pull request:

- Fixed issue where the install instructions left out ngx-quicklink

### CODE REVIEW

**Comment by Alex61NN5** [comment]
I signed it!

**Comment by robdodson** [comment]
Thanks for the patch @Alex61NN5! I'm not an angular expert but @mgechev should be able to confirm if this info is correct.

File: src/site/content/en/angular/route-preloading-in-angular/index.md [hunk 1]
```diff
@@ -72,10 +72,10 @@ The router also registered the modules' route declarations so that when you navi
 
 The [quicklink](https://github.com/GoogleChromeLabs/quicklink) library provides a better strategy for larger apps. It uses the [IntersectionObserver](https://developers.google.com/web/updates/2019/02/intersectionobserver-v2) API to preload only modules associated with links that are currently visible on the page.
 
-You can add quicklink to an Angular app by using the [ngx-quicklink](https://www.npmjs.com/package/ngx-quicklink) package. Start by installing the package from npm:
+You can add quicklink to an Angular app by using the [ngx-quicklink](https://www.npmjs.com/package/ngx-quicklink) package. Start by installing the packages from npm:
 
 ```
-npm install --save quicklink
+npm install --save quicklink ngx-quicklink
```
Comment (mgechev) [inline src/site/content/en/angular/route-preloading-in-angular/index.md]:
```suggestion
npm install --save ngx-quicklink
```

File: src/site/content/en/angular/route-preloading-in-angular/index.md [hunk 2]
```diff
@@ -72,10 +72,10 @@ The router also registered the modules' route declarations so that when you navi
 
 The [quicklink](https://github.com/GoogleChromeLabs/quicklink) library provides a better strategy for larger apps. It uses the [IntersectionObserver](https://developers.google.com/web/updates/2019/02/intersectionobserver-v2) API to preload only modules associated with links that are currently visible on the page.
 
-You can add quicklink to an Angular app by using the [ngx-quicklink](https://www.npmjs.com/package/ngx-quicklink) package. Start by installing the package from npm:
+You can add quicklink to an Angular app by using the [ngx-quicklink](https://www.npmjs.com/package/ngx-quicklink) package. Start by installing the packages from npm:
```
Comment (mgechev) [inline src/site/content/en/angular/route-preloading-in-angular/index.md]:
```suggestion
You can add quicklink to an Angular app by using the [ngx-quicklink](https://www.npmjs.com/package/ngx-quicklink) package. Start by installing the package from npm:
```

**Comment by Alex61NN5** [comment]
I think @mgechev needs to consent to co-authoring

**Pull request merged**